boards/myboard/myfc/default.cmake

```diff
@@ -0,0 +1,105 @@
+px4_add_board(
+PLATFORM nuttx
+VENDOR myboard
+MODEL myfc
+LABEL default
+TOOLCHAIN arm-none-eabi
+ARCHITECTURE cortex-m0plus
+CONSTRAINED_MEMORY
+ROMFSROOT px4fmu_common
+CONSTRAINED_FLASH
+SERIAL_PORTS
+# 	TEL2:/dev/ttyS1
+# 	URT6:/dev/ttyS2
+DRIVERS
+# 	adc/board_adc
+# 	#barometer # all available barometer drivers
+# 	barometer/bmp280
+# 	#batt_smbus
+# 	#camera_trigger
+# 	#differential_pressure # all available differential pressure drivers
+# 	#distance_sensor # all available distance sensor drivers
+# 	dshot
+# 	gps
+# 	imu/invensense/icm20602
+# 	imu/invensense/mpu6000
+# 	#irlock
+# 	lights/rgbled
+# 	#magnetometer # all available magnetometer drivers
+# 	magnetometer/hmc5883
+# 	#optical_flow # all available optical flow drivers
+# 	osd
+# 	#pca9685
+# 	#pwm_input
+# 	#pwm_out_sim
+# 	pwm_out
+# 	rc_input
+# 	#telemetry # all available telemetry drivers
+# 	telemetry/frsky_telemetry
+MODULES
+# 	#airspeed_selector
+# 	attitude_estimator_q
+# 	battery_status
+# 	#camera_feedback
+# 	commander
+# 	dataman
+# 	ekf2
+# 	#esc_battery
+# 	events
+# 	flight_mode_manager
+# 	#fw_att_control
+# 	#fw_pos_control_l1
+# 	gyro_calibration
+# 	#gyro_fft
+# 	land_detector
+# 	#landing_target_estimator
+# 	load_mon
+# 	#local_position_estimator
+# 	logger
+# 	mavlink
+# 	mc_att_control
+# 	mc_hover_thrust_estimator
+# 	mc_pos_control
+# 	mc_rate_control
+# 	#micrortps_bridge
+# 	navigator
+# 	rc_update
+# 	#rover_pos_control
+# 	sensors
+# 	#sih
+# 	#temperature_compensation
+# 	#uuv_att_control
+# 	#uuv_pos_control
+# 	#vmount
+# 	#vtol_att_control
+SYSTEMCMDS
+# 	#bl_update
+# 	dmesg
+# 	dumpfile
+# 	esc_calib
+# 	#gpio
+# 	hardfault_log
+# 	i2cdetect
+# 	led_control
+# 	#mft
+# 	mixer
+# 	#motor_ramp
+# 	motor_test
+# 	#mtd
+# 	nshterm
+# 	param
+# 	perf
+# 	pwm
+# 	reboot
+# 	reflect
+# 	sd_bench
+# 	#serial_test
+# 	#system_time
+# 	top
+# 	#topic_listener
+# 	tune_control
+# 	uorb
+# 	usb_connected
+# 	ver
+# 	work_queue
+)
```

Comment:
There should be some simple things that can be enabled above.

[this board](https://www.seeedstudio.com/ARDUINO-NANO-RP2040-CONNECT-WITH-HEADERS-p-4941.html?gclid=Cj0KCQjw0emHBhC1ARIsAL1QGNf-GtXTRIa8gkXtLtUyuvqL9llB5J3dkLwOqNLLiq6Hgsa44-5NNVYaAqotEALw_wcB) may be a good port to nuttx to get some more feature support. It has an IMU.... 


Some above are hardware dependent, but look for easy wins and it will help create a todo list for nuttx driover support. 

E.G. SD support over SPI. use sd_bench to test.

UUID may be solved with something like https://www.microchip.com/wwwproducts/en/24aa02uid (bigger will be better)  This can also be used to store a small amount of parameters for the param system (Flash can be used as well) but that will be need a program mem nuttx driver. 

ver, top serial_test  should be easy wins.